front end/index.js

```diff
@@ -0,0 +1,89 @@
+const listClasses = async()=>{
+    try{
+        //let select = document.querySelector("#select")
+        let res = await axios.get("http://localhost:7000/class")
+        let classes = res.data.classes
+        let select = document.querySelector("#select")
+        Object.values(classes).forEach(values=>{
+            //debugger
+            let option = document.createElement("option")
+            option.innerText=values.name
+            select.appendChild(option)
+        })
+               
+        
+    }catch(err){
+        console.log(err)
+    }
+}
+listClasses()
+
+const addClass = async()=>{
+    try{
+        let form= document.querySelector("form")
+        form.addEventListener("submit",async(event)=>{
+            event.preventDefault()
+            let classes = document.querySelector("#classes")
+            let teacher = document.querySelector("#teacher")
+            await axios.post("http://localhost:7000/class",{name:classes.value, teacher:teacher.value})
+            let select = document.querySelector("#select")
+            //debugger
+            let option = document.createElement("option")
+            option.innerText = classes.value
```

Comment:
Your options should come from your backend. That way if the class could not be added it will not be appended as an option. Right now if I keep adding the same class over and over again, it'll keep appending as an option to the select.